tests/Php/PhpIniHelperTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright © 2017 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+declare(strict_types=1);
+
+namespace Infection\Tests\Php;
+
+use Infection\Php\PhpIniHelper;
+use PHPUnit\Framework\TestCase;
+
+class PhpIniHelperTest extends TestCase
+{
+    public static $envOriginal;
+
+    public function test_it_works_with_loaded_ini()
+    {
+        $paths = [
+            'test.ini',
+        ];
+
+        $this->setEnv($paths);
+        $this->assertEquals($paths, PhpIniHelper::get());
```

Comment:
`$this->assertSame()` would be preferable here: it is strictly so safer to use it as default and use `assertEquals()` only when appropriate